testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksUnityCompatibility.java

```diff
@@ -0,0 +1,232 @@
+                            row("double_col", "double", "", ""),
+                            row("date_col", "date", "", ""),
+                            row("timestamp_col", "timestamp(3) with time zone", "", ""),
+                            row("binary_col", "varbinary", "", ""),
+                            row("bool_col", "boolean", "", ""),
+                            row("array_int_col", "array(integer)", "", ""),
+                            row("map_col", "map(timestamp(3) with time zone, integer)", "", ""),
+                            row("struct_col", "row(a bigint, b varchar)", "", ""));
+        }
+        finally {
+            onDelta().executeQuery(format("DROP SCHEMA IF EXISTS %s.%s CASCADE", unityCatalogCatalogName, schemaName));
+        }
+    }
+
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testDdl()
+    {
+        String schemaName = "testDdl_" + randomNameSuffix();
+        String tableName = "t1";
+        String schemaLocation = getLocationForSchema(bucketName, externalLocationPath, schemaName);
+        String tableLocation = getLocationForTable(bucketName, externalLocationPath, schemaName, tableName);
+        String trinoLocation = getLocationForSchema(bucketName, externalLocationPath, "trino_test_" + randomNameSuffix());
+        try {
+            onDelta().executeQuery(format("CREATE SCHEMA %s.%s MANAGED LOCATION '%s'",
+                    unityCatalogCatalogName, schemaName, schemaLocation));
+            onDelta().executeQuery(
+                    format("CREATE TABLE %s.%s.%s (c1 int, c2 string) using delta location '%s'",
+                            unityCatalogCatalogName, schemaName, tableName, tableLocation));
+            assertThatThrownBy(
+                    () -> onTrino().executeQuery(format("CREATE TABLE delta.%s.new_table (c1 int) WITH (location = '%s')", schemaName, trinoLocation)))
+                    .hasMessageContaining("DDL not enabled in Hive metastore interface.");
+            assertThatThrownBy(
+                    () -> onTrino().executeQuery(format("CREATE TABLE delta.%s.new_table (c1) WITH (location = '%s') AS SELECT 1", schemaName, trinoLocation)))
+                    .hasRootCauseMessage("DDL not enabled in Hive metastore interface.");
+            assertThatThrownBy(
+                    () -> onTrino().executeQuery(format("ALTER TABLE delta.%s.%s RENAME TO delta.%s.new_t1", schemaName, tableName, schemaName)))
+                    .hasMessageContaining("DDL not enabled in Hive metastore interface.");
+            assertThatThrownBy(
+                    () -> onTrino().executeQuery(format("DROP TABLE delta.%s.%s", schemaName, tableName)))
+                    .hasMessageContaining("DDL not enabled in Hive metastore interface.");
+            /* assertThatThrownBy(
```

Comment:
left-over?

Reply:
This should throw an exception but while testing I found a bug on the Databricks side where create schema is not gated.  I am working on the fixing it and then we can add it enable this in a followup PR.

Reply:
Please create a follow-up issue and remove the commented code.